lib/internal/Magento/Framework/Indexer/Test/Unit/CacheContextTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\Indexer\Test\Unit;
+
+use Magento\Framework\Indexer\CacheContext;
+use PHPUnit\Framework\TestCase;
+
+class CacheContextTest extends TestCase
+{
+    /**
+     * @var Batch
+     */
+    private $object;
+
+    protected function setUp(): void
+    {
+        $this->object = new CacheContext();
+    }
+
+    /**
+     * @param array $tagsData
+     * @param array $expected
+     * @dataProvider getTagsDataProvider
+     */
+    public function testUniqueTags($tagsData, $expected)
```

Comment:
Just a recommendation for the future to use type hinting whenever it's possible. It's a very useful habit in PHP 7.x